### PR TITLE
Maximum/minimum value for Long/TimeSpan

### PR DESCRIPTION
**Pull request by emilwestergren**
* Added min/max value for `Long`
* Added min/max value for `TimeSpan`

### CODE REVIEW

**Comment by emilwestergren** [comment]
@marcusnaslund peer review

File: source/system/math.ooc [hunk 1]
```diff
@@ -11,6 +11,8 @@ use math-lib
 
 FLT_EPSILON: extern Float
 DBL_EPSILON: extern Double
+INT64_MAX: extern Long
+INT64_MIN: extern Long
```
Comment (marcusnaslund) [inline source/system/math.ooc]: These should be defined in `Numbers.ooc` together with the others.

File: source/base/TimeSpan.ooc [hunk 2]
```diff
@@ -103,6 +103,8 @@ TimeSpan: cover {
 	hours: static func (count: Double) -> This { This new(DateTime ticksPerHour * count) }
 	days: static func (count: Double) -> This { This new(DateTime ticksPerDay * count) }
 	weeks: static func (count: Double) -> This { This new(DateTime ticksPerWeek * count) }
+	maximumValue ::= static This new(Long maximumValue)
+	minimumValue ::= static This new(Long minimumValue)
```
Comment (marcusnaslund) [inline source/base/TimeSpan.ooc]: These make more sense above line 93 instead

File: source/system/math.ooc [hunk 3]
```diff
@@ -33,6 +35,8 @@ extend Long {
 	}
 	maximum: func (other: This) -> This { this > other ? this : other }
 	minimum: func (other: This) -> This { this < other ? this : other }
+	maximumValue ::= static INT64_MAX
+	minimumValue ::= static INT64_MIN
```
Comment (marcusnaslund) [inline source/system/math.ooc]: These should be at the top (like in `Int` and others)

**Comment by marcusnaslund** [comment]
Besides changing everything, this looks good.

File: source/system/math.ooc (hunk 3 above)
Comment (emilwestergren) [inline source/system/math.ooc]: I thought static members should always be at the bottom?

Comment (marcusnaslund) [inline source/system/math.ooc]: Hmm generally yeah, except we (meaning I) seem to have always put them at the top in this file. Let's be consistent.

Comment (emilwestergren) [inline source/system/math.ooc]: Sort of breaks the point of the general policy though, but alright I don't mind.

**Comment by emilwestergren** [comment]
Updated

**Comment by marcusnaslund** [comment]
Excellent.

**Pull request merged**